user-console/src/main/java/org/pentaho/mantle/client/commands/SaveCommand.java

```diff
@@ -118,55 +130,111 @@ public void fileSelected( final RepositoryFile file, String filePath, String fil
 
           if ( dialog.doesSelectedFileExist( fileExtension ) ) {
             dialog.hide();
+
             PromptDialogBox overWriteDialog =
               new PromptDialogBox(
                 Messages.getString( "question" ), Messages.getString( "yes" ), Messages.getString( "no" ), //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
                 false, true );
             overWriteDialog.setContent( new Label( Messages.getString( "fileExistsOverwrite" ), false ) ); //$NON-NLS-1$
             overWriteDialog.setCallback( new IDialogCallback() {
               public void okPressed() {
+                // Save as, overwriting existing file.
+
                 if ( fileExtension != null && tabName.endsWith( fileExtension ) ) {
                   tabName = tabName.substring( 0, tabName.lastIndexOf( fileExtension ) );
                 }
+
+                final CommandResultCallback composedCallback = composeCallbacks( new CommandResultCallback() {
+                  @Override
+                  public void onSuccess() {
+                    Window.setTitle( Messages.getString( "productName" ) + " - " + name ); //$NON-NLS-1$ //$NON-NLS-2$
```

Comment:
//$NON-NLS-1$ //$NON-NLS-2$ comments can be removed, they only served to suppress warning in Eclipse

Reply:
I made special effort of not touching more lines than needed. This is old code. It's probably easier to make a global find/replace in a dedicated CLEANUP PR.

Reply:
I can do it, afterwards.